pages/api/fetch-usage-details/index.js

```diff
@@ -15,6 +15,13 @@ async function handler(req, res) {
 async function getProjectUse(packageName, repo, dir = '') {
   const { data, error } = await getPackageJson(repo, dir)
   if (error) return { error }
+  if (!data.dependencies) {
```

Comment:
This fixes a bug if `package.json` isn't present or doesn't have dependencies, as is currently the case since https://github.com/hashicorp/nextjs-scripts has been split up, but we haven't yet updated the list of consuming projects.